ruby_on_rails/vcr.md

```diff
@@ -1,8 +1,56 @@
 # VCR
 
-VCR is a gem for recording HTTP requests to external services and replaying them. A VCR setup should be very specifically tied to
-the tests which need VCR because it's quite expensive.
+VCR is a gem for recording HTTP requests to external services and replaying them.
+A VCR setup should be very specifically tied to the tests which need VCR because it's quite expensive.
 
-Here's an article on about some tweaks which can be made to a VCR setup: https://blog.arkency.com/3-tips-to-tune-your-vcr-in-tests/
+Here's an example configuration we use at Renuo:
+
+```rb
+# spec/rails_helper.rb
+
+RSpec.configure do |config|
+  # …
+
+  # Disable VCR completely for tests that are not tagged with :vcr
+  config.around do |example|
+    if example.metadata[:vcr]
+      example.run
+    else
+      VCR.turned_off { example.run }
+    end
+  end
+end
+
+# WebMock catches everything per default, we allow localhost for Capybara/Selenium
+WebMock.disable_net_connect!(allow_localhost: true)
+
+VCR.configure do |c|
+  c.configure_rspec_metadata!
+  c.debug_logger = $stderr if ENV['DEBUG'] == 'true'
+
+  c.hook_into :webmock
+  c.ignore_localhost = true
+
+  c.cassette_library_dir = 'spec/vcr'
+  c.default_cassette_options = {
+    decode_compressed_response: true,
+    allow_unused_http_interactions: false,
+    record: ENV['VCR'] ? ENV['VCR'].to_sym : :once, # re-record with VCR=all
```

Comment:
```suggestion
    record: ENV.fetch('VCR_RECORD', :once).to_sym, # re-record with VCR_RECORD=all
```

I'd prefer to be a bit more explicit in the variable name since just `VCR=all` can easily be misinterpreted as "have all VCRs active".

Reply:
Even though you're technically correct, I disagree. I could not remember anything longer than `VCR`. For example I like very much that @sihu uses `DEBUG=true` for `selenium_chrome` driver instead of headless. It's very unspecific but very rememberable.